pandas/io/tests/test_clipboard.py

```diff
@@ -113,3 +116,12 @@ def test_read_clipboard_infer_excel(self):
         exp = pd.read_clipboard()
 
         tm.assert_frame_equal(res, exp)
+
+    #test case for testing invalid encoding
+    def test_invalid_encoding(self):
+        data = self.data['string']
+        with tm.assertRaises(ValueError):
+            data.to_clipboard(encoding='ascii')
```

Comment:
right in reading you still need to strip the passed encoding (and read from the clipboard as utc8)

then i guess we could reencode as the encoding (if it's not None)

put a test for this but for now I would just raise NotImplementError if an encoding is passed in and it's not utf8 or none